src/sensai/vector_model.py

```diff
@@ -253,7 +326,11 @@ def __init__(self):
     def isRegressionModel(self) -> bool:
         return False
 
-    def _fit(self, X: pd.DataFrame, Y: pd.DataFrame):
+    def _fit(self, X: pd.DataFrame, Y: Optional[pd.DataFrame]):
+        if Y is None:
+            raise NotImplemented(
+                "Unsupervised classification models are currently not implemented"
+            )
```

Comment:
Continued statements must always use indentation. Please do not use this style of formatting.

Reply:
So you are not a fan of black formatter 😄 . Ok, I will remove it.

The idea is to minimize the diff, so here a change in the string is on a separate line and one immediately see that nothing else has changed

Reply:
Removed this code